pxr/usd/usdSkel/doxygen/apiIntro.dox

```diff
@@ -795,10 +800,10 @@ WriteAnimatedSkel(
     }
 
     UsdSkelAnimation anim = UsdSkelAnimation::Define(
-        stage, skelPath.AppendChild(TfToken("Anim")));
+        stage, skel.GetPath().AppendChild(TfToken("Anim")));
 
     UsdSkelBindingAPI binding = UsdSkelBindingAPI::Apply(skel.GetPrim());
-    binding.CreateSkeletonRel().SetTargets(
+    binding.CreateAnimationSourceRel().SetTargets(
```

Comment:
Bug fix.

Reply:
I could be wrong, but I believe the repeated section of code down near line 1081 needs to be updated to use CreateAnimationSourceRel() instead of CreateSkeletonRel() as you did here (and similarly for the Python snippet). I'm working on merging this change in, so I'll update accordingly on my end.

Reply:
Good catch. I missed the fix in the breakdown section. I went ahead and updated it in a new commit along with another suggestion from Spiff on the AOUSD forum.